plop/collector.py

```diff
@@ -182,18 +183,19 @@ def main():
                 # interpreted as relative to this module.
                 global __package__
                 del __package__
-                exec f.read() in globals(), globals()
-    except SystemExit, e:
+                exec(f.read() in globals(), globals())
```

Comment:
Fixed, but i don't know why not always but sometimes running `python plop/test/runtests.py` is failed even in main master code!
Here is output of failed test:
```
[W 150904 00:17:32 collector_test:38] ('a', 'test_collect_threads'): expected 10, got 10 (1.0)
[W 150904 00:17:32 collector_test:38] ('a', 'thread1_func'): expected 20, got 31 (1.55)
[W 150904 00:17:32 collector_test:38] ('a', 'thread2_func'): expected 30, got 38 (1.26666666667)
[W 150904 00:17:32 collector_test:40] unexpected key: ('test_collect_threads',): got 10
F.....
======================================================================
FAIL: test_collect_threads (plop.test.collector_test.CollectorTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/***/env2.7/local/lib/python2.7/site-packages/plop/test/collector_test.py", line 109, in test_collect_threads
    self.check_counts(counts, expected)
  File "/***/env2.7/local/lib/python2.7/site-packages/plop/test/collector_test.py", line 41, in check_counts
    self.fail("collected data did not meet expectations")
AssertionError: collected data did not meet expectations
----------------------------------------------------------------------
Ran 6 tests in 1.686s

FAILED (failures=1)
[E 150904 00:17:34 testing:697] FAIL
```